cli/cli.go

```diff
@@ -65,59 +64,148 @@ func (a *Application) init() {
+				Subcommands: []*cli.Command{
+					{
+						Name:   "stats",
+						Usage:  "print your stats",
+						Before: a.initApiConnection,
+						Action: func(c *cli.Context) error {
+							return printStatus(a.api)
+						},
+					},
+					{
+						Name:   "id",
+						Usage:  "print yours peer id",
+						Before: a.initApiConnection,
+						Action: func(c *cli.Context) error {
+							return printPeerId(a.api)
+						},
+					},
+				},
+			},
+			{
+				Name:  "peers",
+				Usage: "group of functions to work with peers. Use for check friend requests and peers status",
+				Subcommands: []*cli.Command{
+					{
+						Name:  "status",
+						Usage: "print peers status",
+						Flags: []cli.Flag{
+							&cli.StringFlag{
+								Name:     "format",
+								Aliases:  []string{"f"},
+								Required: false,
+								Value:    "npsdaltrcv",
+								Usage: "control table columns list and order.Each char add column, write column chars together without gap. Use these chars to add specific columns:\n   " +
+									"n - peers number\n   p - peers name\n   i - peers id\n   s - peers status\n   d - peers domain\n   a - peers ip address\n   l - peers last seen datetime\n   v - peers awl version" +
+									"\n   t - total network usage by peer (in/out)\n   r - network usage speed by peer (in/out)\n   c - list of peers connections (IP address + protocol)\n  ",
+							},
+						},
+						Before: a.initApiConnection,
+						Action: func(c *cli.Context) error {
+							return printPeersStatus(a.api, c.String("format"))
+						},
+					},
+					{
+						Name:   "requests",
+						Usage:  "print all incoming friend requests",
+						Before: a.initApiConnection,
+						Action: func(c *cli.Context) error {
+							return printFriendRequests(a.api)
+						},
+					},
+				},
+			},
+			{
+				Name:  "peer",
+				Usage: "group of commands to work with peer (one of the flags is required)",
 				Flags: []cli.Flag{
 					&cli.StringFlag{
-						Name:     "peer_id",
-						Required: true,
+						Name:     "pid",
+						Usage:    "peer id",
+						Required: false,
 					},
 					&cli.StringFlag{
 						Name:     "alias",
 						Required: false,
 					},
 				},
-				Before: a.initApiConnection,
-				Action: func(c *cli.Context) error {
-					peerID := c.String("peer_id")
-					alias := c.String("alias")
-
-					authRequests, err := a.api.AuthRequests()
+				Before: func(c *cli.Context) error {
+					err := a.initApiConnection(c)
 					if err != nil {
 						return err
 					}
-					hasRequest := false
-					for _, req := range authRequests {
-						if req.PeerID == peerID {
-							hasRequest = true
-							break
-						}
+
+					pid := c.String("pid")
+					if pid != "" {
+						return nil
 					}
-					if hasRequest {
-						err := a.api.ReplyFriendRequest(peerID, alias, false)
-						return err
+					alias := c.String("alias")
+					if alias == "" {
+						return fmt.Errorf("peerID or alias should be defined")
```

Comment:
You need to check this in Action code (or check the presence of `--help`). Otherwise, you can't use `--help`.

```
$ sudo ./awl-tray cli peer ch_alias --help
2022-10-20T23:59:50.761+0300    FATAL   awl/cli Error occurred: peerID or alias should be defined
```